RevenueCatUI/Templates/V2/Components/Image/ImageComponentView.swift

```diff
@@ -48,24 +51,48 @@ struct ImageComponentView: View {
                 darkUrl: style.darkUrl,
                 darkLowResUrl: style.darkLowResUrl
             ) { (image, size) in
-                renderImage(image, size, with: style)
+                self.renderImage(image, size, with: style)
             }
             .size(style.size)
```

Comment:
One thing we did in Android was that, if the size is `fit` in any axis, we change it to`Fixed` with the given backend sizes. Not sure if we want to do the same thing here?

Reply:
@tonidero What if the parent/container is smaller than what the image is? Wouldn't the fixed make the image too big then? 🤔

Reply:
That's a good thing to test, but I don't think so. We didn't use `requiredSize()` (which would ignore constraints set by the parent), so if the parent is smaller, the image will be sized down. (At least that's what I expect.)

Reply:
Yup, can confirm it will be constrained to the available space, but will add some previews so we have snapshot tests for that as well 👍